bgpd/bgp_route.c

```diff
@@ -11933,10 +11942,11 @@ DEFPY (show_ip_bgp_json,
 	uint8_t show_flags = 0;
 	enum rpki_states rpki_target_state = RPKI_NOT_BEING_USED;
 
-	if (uj) {
-		argc--;
```

Comment:
why is this line removed ?

Reply:
Fixed. But not sure why is it used here.